src/inmanta/ast/statements/generator.py

```diff
@@ -242,32 +244,47 @@ def __init__(
         self,
         class_type: LocatableString,
         attributes: List[Tuple[LocatableString, ExpressionStatement]],
+        wrapped_kwargs: List["WrappedKwargs"],
         location: Location,
         namespace: Namespace,
     ) -> None:
         super().__init__()
         self.class_type = str(class_type)
         self.__attributes = {}  # type: Dict[str,ExpressionStatement]
+        self.__wrapped_kwarg_attributes: List[WrappedKwargs] = wrapped_kwargs
         self.location = location
         self.namespace = namespace
         self.anchors.append(TypeReferenceAnchor(class_type.get_location(), namespace, str(class_type)))
         for a in attributes:
             self.add_attribute(a[0], a[1])
+        self.type: Optional["EntityLike"] = None
+        self.required_kwargs: List[str] = []
 
         self._direct_attributes = {}  # type: Dict[str,ExpressionStatement]
         self._indirect_attributes = {}  # type: Dict[str,ExpressionStatement]
 
     def pretty_print(self) -> str:
-        return "%s(%s)" % (self.class_type, ",".join(("%s=%s" % (k, v.pretty_print()) for k, v in self.attributes.items())))
+        # TODO: test case for this pretty_print
```

Comment:
...

Reply:
oops, usually I grep for TODO's before starting a pull request, I must've forgotten. Is such a test actually useful? I didn't encounter similar ones so I opted against.